apps/sertra/components/mod-selector/modal.module.css

```diff
@@ -25,66 +25,64 @@
 }
 
 .header {
-  background-color: var(--bg-light);
   display: flex;
   flex-direction: column;
-  padding: 24px 32px 16px;
+  padding: 1.5rem 2rem 1rem;
+  background-color: var(--bg-light);
 }
 
 .headerTitle {
   display: flex;
-  flex-direction: row;
+  flex-direction: row-reverse;
```

Comment:
Is there a reason the for the close-button and title being in reverse order in the DOM compared to the rendered output, thus needing this?

Reply:
We did this change during the pair (group) work session on Thursday and actually shortly pondered the ramifications of having the close button before the title on the source order. We didn't come up with any reasons why this would be a bad thing and decided to go with it. But basically this is because in the component the title is optional (for some reason) and it's easier to "fix" the close button to right this way. So, what are the cons in this approach (besides it being mildly hacky)?